src/Mount.cpp

```diff
@@ -1242,25 +1257,33 @@ const Declination Mount::currentDEC() const
 void Mount::syncPosition(DayTime ra, Declination dec)
 {
     long solutions[6];
-    _targetRA  = ra;
     _targetDEC = dec;
+    _targetRA  = ra;
+
+    // Adjust the home RA position by the delta sync position.
+    float raAdjust = ra.getTotalHours() - currentRA().getTotalHours();
+    while (raAdjust > 12)
+    {
+        raAdjust = raAdjust - 24;
+    }
+    while (raAdjust < -12)
+    {
+        raAdjust = raAdjust + 24;
+    }
+    _zeroPosRA.addHours(raAdjust);
+
+    // Adjust the home DEC position by the delta between the sync'd target and current position.
+    float decAdjust = dec.getTotalDegrees() - currentDEC().getTotalDegrees();
+    _zeroPosDEC += decAdjust;
 
     long targetRAPosition, targetDECPosition;
-    LOGV3(DEBUG_MOUNT, F("[MOUNT]: Sync Position to RA: %s and DEC: %s"), _targetRA.ToString(), _targetDEC.ToString());
+    LOGV3(DEBUG_MOUNT, F("[MOUNT]: Sync Position to RA: %s and DEC: %s"), ra.ToString(), _targetDEC.ToString());
```

Comment:
Shouldn't this log be both `ra, dec` or `_targetRA, _targetDEC` and not one of either?